AK/IntrusiveList.h

```diff
@@ -320,8 +320,7 @@ namespace Detail {
 template<typename T, typename Container>
 inline IntrusiveListNode<T, Container>::~IntrusiveListNode()
 {
-    if (m_storage)
-        remove();
+    VERIFY(m_storage == nullptr);
```

Comment:
Nitpick: Should we just use `IntrusiveListNode::is_in_list()`?
```suggestion
    VERIFY(!is_in_list());
```

Reply:
Fixed.